tests/test_envs.py

```diff
@@ -16,7 +16,10 @@
     if env_spec.id.startswith(f"{seals.GYM_ID_PREFIX}/")
 ]
 
-DETERMINISTIC_ENVS: List[str] = []
+DETERMINISTIC_ENVS: List[str] = [
```

Comment:
On reflection I wonder if it would be better for environments to declare whether they are deterministic or stochastic as an attribute. Thoughts @qxcv? Probably not worth addressing in this PR.